conda/plan.py

```diff
@@ -64,10 +63,10 @@ def display_actions(actions, index, show_channel_urls=None):
         print_dists(disp_lst)
 
         if index and len(actions[inst.FETCH]) > 1:
+            num_bytes = sum(index[dist + '.tar.bz2']['size']
+                            for dist in actions[inst.FETCH])
```

Comment:
Could have been the same line, as far as I'm concerned.  (Also don't mind it being two lines.)  I'm sure you noticed max line length set to 99 in `setup.cfg`.  That was definitely intentional.  Still PEP-8 compliant, but for me, more comfortable than 79.